src/ChineseNumbers/ChineseNumbers.h

```diff
@@ -0,0 +1,15 @@
+#ifndef FCITX5_MCBOPOMOFO_CHINESENUMBER_H
+#define FCITX5_MCBOPOMOFO_CHINESENUMBER_H
+
+#include <string>
+namespace ChineseNumbers {
+enum class ChineseNumberCase { LOWERCASE, UPPERCASE };
+
+namespace ChineseNumbers {
```

Comment:
Why nested namespace? Wouldn't the enum `ChineseNumbers::ChineseNumbercase` and the function `ChineseNumbers::Generate(...)` already suffice?

Reply:
Fixed in 92e46601d055fb57699128d5d2cb8634a1fef339